### PR TITLE
Tweak

### PR DESCRIPTION
**Pull request by JedMeister**
Tweaks to tkldev-detective to get the package installable and working.

Happy to further discuss my code changes to tkldev-detective @OnGle. E.g. if you want rationale for anything and/or are unhappy with anything.

Note that this code is currently installable from the `bookworm-testing` repo. But be sure to remove the default `tkldev-changelog` in v18.0rc1 (`rm /usr/local/bin/tkldev-changelog`). It will be removed for v18.0 proper.

### CODE REVIEW

File: libtkldet/modman.py [hunk 1]
```diff
@@ -19,11 +19,15 @@
...
-from os.path import join, dirname, abspath, splitext, isfile
+from os.path import join, dirname, abspath, splitext, isfile, exists
 from os import listdir
 import sys
 
 from . import colors as co
+from .error import TKLDevDetectiveError
+
+# priortise local tkldet_modules path, fallback to OS path
+MOD_PATH = [ abspath(__file__), '/usr/share/tkldev-detective']
```
Comment (OnGle) [inline libtkldet/modman.py]: pretty sure it should be `dirname(dirname(abspath(__file__)))` not just `abspath(__file__)`

**Review by OnGle — CHANGES_REQUESTED** [review]

File: libtkldet/modman.py (hunk 1 above)
Comment (JedMeister) [reply libtkldet/modman.py]: Thanks @OnGle, good catch!

I didn't actually test that, so you're probably right. I'll double check right now and report back.

Comment (JedMeister) [reply libtkldet/modman.py]: Bug confirmed, and fixed!

**Comment by JedMeister** [comment]
@OnGle - I also applied the improvement you suggested offline to `libtkldet/apt_file.py`.

**Review by OnGle — APPROVED** [review]
lgtm

**Review by Christie018 — APPROVED** [review]
Code changes look good to me 👍🏾 Not sure if tracker issue [#1116](https://github.com/turnkeylinux/tracker/issues/1116) is relevant here but I noticed there was still an issue open related to composer so I thought I'd bring it up.

**Comment by JedMeister** [comment]
> Not sure if tracker issue https://github.com/turnkeylinux/tracker/issues/1116 is relevant here but I noticed there was still an issue open related to composer so I thought I'd bring it up.

Excellent suggestion Mattie! :)

I don't think that it explicitly relevant here, but your suggestion did prompt me to improve that issue a little and give some more context. I'm not going to do anything with it yet, but with the additional info the point of the issue is now perhaps a bit clearer?!

**Pull request merged**